stable.json

```diff
@@ -25,22 +25,22 @@
     "odroid-xu": "2023.6.2"
   },
   "hassos": {
-    "ova": "10.2",
-    "rpi2": "10.2",
-    "rpi3": "10.2",
-    "rpi3-64": "10.2",
-    "rpi4": "10.2",
-    "rpi4-64": "10.2",
-    "yellow": "10.2",
-    "tinker": "10.2",
-    "odroid-c2": "10.2",
-    "odroid-c4": "10.2",
-    "odroid-m1": "10.2",
-    "odroid-n2": "10.2",
-    "odroid-xu4": "10.2",
-    "generic-x86-64": "10.2",
-    "generic-aarch64": "10.2",
-    "khadas-vim3": "10.2"
+    "ova": "10.3",
```

Comment:
<https://github.com/home-assistant/operating-system/releases/tag/10.3> 
> ℹ️ ℹ️ ℹ️ This release is currently only on the beta channel ℹ️ ℹ️ ℹ️

Reply:
It still is :wink:

Reply:
Updated, thanks :+1: